Lib/fontTools/feaLib/lexer.py

```diff
@@ -193,14 +195,14 @@ def __init__(self, featurefile):
     def __iter__(self):
         return self
 
-    def next(self):  # Python 2
-        return self.__next__()
+    def next(self, comments = False):  # Python 2
```

Comment:
wow yes very nit picky. I hadn't realise just how nit picky for PEP8 this code base is. Perhaps we need a PEP8 reviewer in the continuous integration testing. I think there is one around somewhere.